bin/gwdetchar-lasso-correlation

```diff
@@ -654,6 +654,18 @@ if args.cluster is True:
     pool = multiprocessing.Pool(nprocplot)
     clusters = pool.map(generate_clusters, enumerate(results[0:len(useful)]))
 
+channelsFile = '%s-CHANNELS-%s.txt'% (args.ifo, gpsstub)
+
+try:
+    numpy.savetxt(channelsFile, channels, fmt="%s")
+except(IOError, IndexError):
+    numpy.savetxt(channelsFile, channels, fmt="%s")
+except RuntimeError as e:
+    if 'latex' in str(e).lower():
```

Comment:
@macedo22, why would latex get involved here?

Reply:
@duncanmmacleod Not sure. I am not too familiar with Latex and because this exception handling was used for saving all of the plots, I assumed it would go here too. I figured the handling was to account for characters in strings that Latex would get mad at, but it sounds like what you're saying is a Latex exception wouldn't be raised from saving channel names to a text file, as it could for images. I will remove the if/else conditional here and just have 
```except RuntimeError:
       raise```

Reply:
there should be no need for any exception handling here at all, so I recommend you remove all of it, unless you understand a configuration where the file writing should be allowed to fail.